browse/services/database/models.py

```diff
@@ -213,10 +216,30 @@ class User(db.Model):
                              index=True, server_default=text("''"))
     flag_allow_tex_produced = Column(
         Integer, nullable=False, server_default=text("'0'"))
-
+    flag_can_lock = Column(Integer,
```

Comment:
good catch

Reply:
Points to the need for a shared legacy-db library.